src/DeploymentSafetyEnforcer.function.ts

```diff
@@ -550,8 +550,9 @@ const execute = async (
       requestId,
       pipelineName: settings.pipelineName,
       rejectedBakeActions: rejectBakeActions.length,
-      settings: settings.metricsSettings ?? {
+      settings: {
         namespace: 'DeploymentSafetyEnforcer',
+        ...settings.metricsSettings,
```

Comment:
What does "Start a review" do

Reply:
Ah, interesting, it puts the comment into "draft" mode, and then when I'm done it'll publish all at once. Neat!